.github/workflows/ci.yml

```diff
@@ -240,6 +240,10 @@ jobs:
           save-cache: ${{ github.ref_name == 'main' }}
           tools: just
 
+      - name: Update transformer fixtures
+        if: steps.filter.outputs.src == 'true'
+        run: bun tasks/transform_conformance/update_fixtures.js
```

Comment:
I removed bun from the main branch, we should be using node.

Reply:
Now using NodeJS.